homeassistant/helpers/config_validation.py

```diff
@@ -571,6 +571,20 @@ def uuid4_hex(value):
     return result.hex
 
 
+def uuid4_string(value):
```

Comment:
We shouldn't add new validators without a use case. The `uuid4_hex` works for all our use cases.

Reply:
I have a use case for the new integration I currently have on review (#28661). Currently I am validating just against strings, but I'd rather have a better validation.